sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -162,6 +173,24 @@ def __init__(self, **kwargs: Any) -> None:
 
             collect_statsbeat_metrics(self)
 
+        # Initialize customer statsbeat if enabled
+        self._customer_statsbeat_metrics = None
+        if self._should_collect_customer_statsbeat():
+            try:
+                statsbeat_options = type('StatsbeatOptions', (), {
+                    'instrumentation_key': self._instrumentation_key,
+                    'endpoint_url': self._endpoint,
+                    'network_collection_interval': kwargs.get('statsbeat_interval', 900000)
+                })
+                # Use the get_customer_statsbeat_metrics function to avoid circular imports
+                CustomerStatsbeatMetricsClass = get_customer_statsbeat_metrics()
+                self._customer_statsbeat_metrics = CustomerStatsbeatMetricsClass(statsbeat_options)
+                # Connect storage with customer statsbeat if storage exists
+                if self.storage:
+                    self.storage._customer_statsbeat_metrics = self._customer_statsbeat_metrics
+            except Exception as ex:
```

Comment:
I think you should abstract out all the initialization logic into customer statsbeat module so that `_base.py` does not need to know anything about it (like how we did for `collect_statsbeat_metrics`).